src/main/java/hanieum/conik/adapter/member/webapi/MemberController.java

```diff
@@ -0,0 +1,68 @@
+package hanieum.conik.adapter.member.webapi;
+
+import hanieum.conik.application.member.MemberModifyService;
+import hanieum.conik.domain.address.dto.AddressRegisterRequest;
+import hanieum.conik.domain.member.dto.MemberProfileUpdateRequest;
+import hanieum.conik.domain.member.dto.PasswordChangeRequest;
+import hanieum.conik.global.adapter.security.CurrentUser;
+import hanieum.conik.global.apiPayload.response.ApiResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1/member")
+@RequiredArgsConstructor
+@Validated
+@Tag(name = "MEMBER", description = "회원 정보를 조회하고 수정하는 API")
+public class MemberController {
+
+    private final MemberModifyService memberModifyService;
+
+    @Operation(summary = "회원 정보 수정", description = """
+    ## 회원 정보 수정을 수행합니다.
+    - 사용자의 전화번호를 입력으로 받아 수정합니다. => 전화번호 인증 로직도 추가할 예정입니다. 
+    """)
+    @PutMapping("/profile")
+    public ApiResponse<?> updateMemberProfile(
+            @CurrentUser Long memberId,
+            @RequestBody @Valid MemberProfileUpdateRequest request
+    ) {
+        memberModifyService.updateProfile(memberId, request);
+        return ApiResponse.success();
+    }
```

Comment:
멤버의 아이디를 가져오기 위함이라면 Spring Security에서 기본적으로 제공되는 @AuthenticationPrincipal를 쓰는 것도 고려해볼 수 있을 것 같아요.

```suggestion
    @PutMapping("/profile")
    public ApiResponse<?> updateMemberProfile(@AuthenticationPrincipal AuthDetails authDetails,
                                              @RequestBody @Valid MemberProfileUpdateRequest request) {
        memberModifyService.updateProfile(authDetails.getMemberId(), request);
        return ApiResponse.success();
    }
```
[참고 1](https://velog.io/@nuh__d/AuthenticationPrincipal%EB%A5%BC-%EC%9C%84%ED%95%9C-Custom-HandlerMethodArgumentResolver)
[참고 2](https://velog.io/@subbni/Spring-Security-AuthenticationPrincipal-CurrentUser)